login.php

```diff
@@ -0,0 +1,98 @@
+<?php
+include('config/config.php');
+if ($enableLogin === true) {
+    if (isset($_POST['submit_updatePwd'])) {
+        if (!empty($_POST["password"]) && ($_POST["password"] == $_POST["repassword"])) {
+            $passwordErr = '';
+            $password = $_POST["password"];
+            
+            if (strlen($_POST["password"]) <= '5') {

```

Comment:
Doesnt make sense. Says Less than or equal to 5 and the error says 8?

Reply:
Solved. Forgot to update the error message. Min character set to 6.